Jenkinsfile

```diff
@@ -114,21 +96,20 @@ pipeline {
       }
 
       environment {
+        GITHUB_TOKEN = credentials('github-api-token')
+        NPM_TOKEN = credentials('npm-publish-token')
         NPM_CONFIG_CACHE = '.npm'
         NPM_CONFIG_USERCONFIG = '.npm/rc'
         SPAWN_WRAP_SHIM_ROOT = '.npm'
-        GITHUB_PACKAGES_TOKEN = credentials('github-api-token')
+        GIT_BRANCH = "${CURRENT_BRANCH}"
+        BRANCH_NAME = "${CURRENT_BRANCH}"
+        CHANGE_ID = ""
```

Comment:
Is `CHANGE_ID` being used anywhere or do we just to unset its value?

Reply:
We need to just unset it.  This was from some weirdness with how Jenkins does PRs from various sources.